components/FlyForm.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import { StyleSheet, Text, View, TextInput, TouchableOpacity } from 'react-native';
+
+class FlyForm extends Component {
+  state = {
+    name: '',
+    color: '',
+    size: '',
+    type: '',
+    amount: null
+  }
+
+  handleName = (text) => {
```

Comment:
We could maybe combine this to be more dynamic? I'm not sure if React Native lets you do something similar to React with using a name property from the input to set state. 

For example, something like this?
```
handleChange = (event) => {
this.setState({ [event.target.name] = event.target.value })
```

Again, not sure if that's possible with React Native or not.

Reply:
As of this moment, this is just the basic setup to get things running.  We can always refactor these codes to one function as we continue on this work on the form. Also, I am not sure React Native have event.target, but we can figure it out when we refactor this section in a future update.